tests/test_asb_save_test_result.yml

```diff
@@ -0,0 +1,38 @@
+---
+- hosts: localhost
+  vars:
+    result_file_path: /var/tmp/test-result
+  roles:
+    - ansibleplaybookbundle.asb-modules
+  tasks:
+    - name: prepare test for clean run
+      file:
+        path: "{{ result_file_path }}"
+        state: absent
+
+    - name: encode bind credentials
+      asb_save_test_result:
+        fail: true
+        msg: "Test failed"
+
```

Comment:
I had thought that we would want to overwrite. But I think your right, that multiple results should be saved.